src/System.Drawing.Common/tests/IconTests.cs

```diff
@@ -385,7 +385,25 @@ public void XpIcon_ToBitmap_Success(int size)
         [ConditionalFact(Helpers.IsDrawingSupported)]
         public void ExtractAssociatedIcon_FilePath_Success()
         {
-            using (Icon icon = Icon.ExtractAssociatedIcon(Helpers.GetTestBitmapPath("48x48_multiple_entries_4bit.ico")))
+            ExtractAssociatedIcon_FilePath_Success(Helpers.GetTestBitmapPath("48x48_multiple_entries_4bit.ico"));
+        }
+
+        [ActiveIssue(20884, TestPlatforms.AnyUnix)]
```

Comment:
[SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework)] // Fix for #34122

Reply:
done thank's!